packages/pharos-site/static/guidelines/toggle-button-group.docs.mdx

```diff
@@ -0,0 +1,143 @@
+import PageSection from '../../src/components/statics/PageSection.tsx';
+import BestPractices from '../../src/components/statics/BestPractices.tsx';
+
+<PageSection
+  isHeader
+  title="Toggle Button Group"
+  storyBookType="components"
+  description="Toggle button groups present related actions together to allow the user to select a mutually exclusive option."
+></PageSection>
+
+<PageSection title="Usage">
+  <p>
+    Within a toggle button group, only one option can be selected at the time. Selecting one option
+    deselects all others.
+  </p>
+  <p>
+    Use a toggle button group as a switch between different views. For example, use the component to
+    allow users to switch between a list view and gallery view.
+  </p>
+</PageSection>
+
+<PageSection title="Best practices">
+  <BestPractices
+    Do={
+      <ul>
+        <li>
+          Items within a toggle button group should be related and have a logical connection to each
+          other
```

Comment:
This item lacks parallel structure with the others, which finish the sentence "Do ________". Maybe this would work? I think we don't have solid parallel structure for our dos/don'ts, so maybe I'm overthinking at this particular juncture.

```suggestion
          Group items within a toggle button group that are related and have a logical connection to each
          other
```